openapi/platform/platform.yaml

```
@@ -2898,6 +2898,47 @@ paths:
           description: A Bearer access token to identify the user the app is acting on behalf of. See the Authorization guide for details.
           name: Authorization
           required: true
+      requestBody:
+        content:
+          application/json:
```

You will need to manually edit this as it does not show up in the dropdown

```suggestion
          application/vnd+api.json:
```